### PR TITLE
init 1 project

### PR DESCRIPTION
**Pull request by AhmadNaruto**

### CODE REVIEW

**Comment by AhmadNaruto** [comment]
sip

File: builds/runtimes/python-3.10.0 [hunk 1]
```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+# Build Path: /app/.heroku/python/
+# Build Deps: libraries/sqlite
+
+OUT_PREFIX=$1
+
+echo "Building Python..."
+SOURCE_TARBALL='https://www.python.org/ftp/python/3.10.0/Python-3.10.0.tgz'
+curl -L $SOURCE_TARBALL | tar xz
+mv Python-3.4.2 src
```
Comment (afaikafk) [inline builds/runtimes/python-3.10.0:10]: should this also be changed to 3.10.0 ?

**Comment by afaikafk** [comment]
thanks for taking the time to create this pull request, but as you can see this repository has not been maintained for a veeery long time. the upstream repo from which I forked has received a large number of updates since then, I would recommend to just take the necessary snippets from here and create a new fork based on the original upstream repo, as there might even be security vulnerabilities in this one, I really don't know, haven't followed upstream development.

**Comment by AhmadNaruto** [comment]
👍  owh.. okay...

**Pull request closed without merging**